modules/oe_content_event/config/install/field.field.node.oe_event.oe_event_online_link.yml

```diff
@@ -18,5 +18,5 @@ default_value: {  }
 default_value_callback: ''
 settings:
   link_type: 17
-  title: 1
+  title: 2
```

Comment:
From the spec
<img width="1107" alt="Screenshot 2021-12-07 at 20 51 15" src="https://user-images.githubusercontent.com/1183951/145080829-bcb54cec-ab67-47fd-95b7-a99960a57e8d.png">